test/FastTests/Issues/RavenDB-9479.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+
+namespace FastTests.Issues
+{
+    public class RavenDB_9479 : RavenTestBase
+    {
+        [Theory]
+        [InlineData((char)0)]
+        [InlineData((char)1)]
+        [InlineData((char)16)]
+        [InlineData((char)32)]
+        [InlineData((char)127)]
+        [InlineData((char)159)]
+        [InlineData((char)114)]
+        public async Task GetStoreAndGetChar(char ch)
+        {
+            using (var store = GetDocumentStore())
+            {
+                string documentId;
+                using (var session = store.OpenAsyncSession())
+                {
+                    var question = new Question
+                    {
+                        Letter = ch
+                    };
+
+                    await session.StoreAsync(question);
+                    await session.SaveChangesAsync();
+
+                    documentId = question.Id;
+                }
+
+                WaitForUserToContinueTheTest(store);
```

Comment:
Should we commit with this?

Reply:
Ideally, not. But if a debugger is not attached this is a noop